src/jobs/utils.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/Arriven/db1000n/src/utils/templates"
 )
 
-func logJob(ctx context.Context, logger *zap.Logger, globalConfig GlobalConfig, args Args) (data interface{}, err error) {
+func logJob(ctx context.Context, logger *zap.Logger, _ GlobalConfig, args Args) (data interface{}, err error) { //nolint:unparam // data is here to match Job
```

Comment:
I wonder why this nolint is needed here but not in other jobs that never return any data?

Reply:
Good question. I have more questions to unparam because it was enabled before this PR, but never complained here. Somehow it started complaining while I was fixing cyclop.

Reply:
Guess we can tackle it separately